src/main/java/com/duffel/service/OfferPassengerService.java

```diff
@@ -0,0 +1,19 @@
+package com.duffel.service;
+
+import com.duffel.model.OfferCollection;
+import com.duffel.model.OfferPassenger;
+import com.duffel.model.request.PostData;
+import com.duffel.net.ApiClient;
+
+public class OfferPassengerService extends PostResource<OfferPassenger, OfferCollection> {
+
+    private static final String ENDPOINT = "/air/offers";
+
+    public OfferPassengerService(ApiClient apiClient) {
+        super(apiClient, ENDPOINT);
+    }
+
+    public OfferPassenger patch(String id, String offerPassengerId, OfferPassenger offerPassenger) {
```

Comment:
Could maybe be more specific about the `id` here.. Is that the `offerId`?

Reply:
Ooh thanks, will do yep